application/SettingsPage.qml

```diff
@@ -22,42 +22,109 @@ import QtQuick.Controls 2.2 as Controls
 import org.kde.kirigami 2.4 as Kirigami
 import Mycroft 1.0 as Mycroft
 
-Kirigami.ScrollablePage {
+Kirigami.Page {
     title: "Settings"
     objectName: "Settings"
     
-
-    ColumnLayout {
+    Column {
         id: settingsLayout
         anchors.fill: parent
-        Layout.fillWidth: true
-        width: parent.width
+        spacing: Kirigami.Units.largeSpacing
+        
+        Kirigami.Heading {
+            id: websocketLabel
+            level: 2
+            font.bold: true
+            color: Kirigami.Theme.textColor;
+            width: parent.width
+            text: "Websocket Address"
+        }
         
         Controls.Label {
-            id: websocketLabel 
-            text: "<h1>Websocket Address</h1> Example: <tt>ws://192.168.1.1</tt>"
+            id: exampleLabel
+            text: "Example: <tt>ws://192.168.1.1</tt>"
+            width: parent.width
         }
         
-        Controls.TextField {
-            id: webSocketAddressField
+        Rectangle {
```

Comment:
why a rectangle?

Reply:
if a background is needed, prefer doing: 
Controls.Control{
contentItem: Controls.TextField{...}
background: Rectangle{...}
}
with this all size hints are automatically propagated correctly
}